jvm/src/main/scala/io/glutenproject/execution/HashAggregateExecTransformer.scala

```diff
@@ -415,11 +415,19 @@ case class HashAggregateExecTransformer(
       val childrenNodeList = new util.ArrayList[ExpressionNode]()
       val childrenNodes = aggExpr.mode match {
         case Partial =>
-          aggregatFunc.children.toList.map(expr => {
-            val aggExpr: Expression = ExpressionConverter
-              .replaceWithExpressionTransformer(expr, originalInputAttributes)
-            aggExpr.asInstanceOf[ExpressionTransformer].doTransform(args)
-          })
+          if (!GlutenConfig.getConf.isClickHouseBackend) {
```

Comment:
Will we move the code related to certain backend only to that specific module in the future?

Reply:
yes, I will optimize these logic codes later.